specification/dsdl/serialization.tex

```diff
@@ -119,16 +119,16 @@ \subsubsection{Implicit truncation of excessive data}
 
 \subsubsection{Implicit zero extension of missing data}
 
-For the purposes of deserialization routines,
-the serialized representation of any instance of any data type shall be considered to contain an implicit
-sequence of trailing zero (0) bits of infinite length.
+For the purposes of deserialization routines, if any received message is shorter than the known datatype an
+implementation shall attempt to accept this message by filling the missing data with zeros (0)\footnote{%
+    When accepting multi-part transfers the subsequent evaluation of the checksum may fail in such cases.
```

Comment:
How about this?

```suggestion
For the purposes of deserialization routines, if any received message is shorter than the known datatype an
implementation shall attempt to accept this message in a way that does not preclude integrity checking on
received multi-frame transfers\footnote{%
    This can be implemented by checking for out-of-bounds access during deserialization and returning zeros
    if an out-of-bounds access is detected. This is where the name "implicit zero extension rule" is derived
    from.
}.
```

Reply:
Maybe it's best to go back to the original phrasing? I think this new wording you proposed is too involved with the transport layer, which is bad because in this chapter we're talking about DSDL. We could keep your footnote though.

If you want to keep the footnote, please fix the quotes:

    ``implicit zero extension rule''

or `\emph{implicit zero extension rule}`.